.github/agents/reporting.agent.md

```
@@ -0,0 +1,294 @@
+### Analysis Reports
+
+**Purpose**: Document code analysis, security findings, audit results
+
+**Location**: `.github/reports/analysis/ANALYSIS-{focus}-{date}.md`
+
+**Examples**:
+- "Analyze the block editor implementation for performance issues"
+- "Audit security in field binding implementation"
+- "Code quality analysis of repeater field handling"
+
+### Migration Reports
```

markdownlint-cli2 (0.18.1)

Multiple headings with the same content

(MD024, no-duplicate-heading)